packages/babel-plugin-polyfill-corejs3/src/index.ts

```diff
@@ -219,6 +219,9 @@ export default defineProvider<Options>(function (
         if (!path.isMemberExpression()) return;
         if (!path.isReferenced()) return;
         if (path.parentPath.isUpdateExpression()) return;
+        if (path.isMemberExpression() && t.isSuper(path.node.object)) {
```

Comment:
```suggestion
        if (t.isSuper(path.node.object)) {
```
`path` must be a member expression at this point.